### PR TITLE
Add Event type visibility check box on internal admin ui

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

File: admin-console/src/main/webapp/src/components/EventTypes/CreateEditModal.tsx [hunk 1]
```diff
@@ -109,6 +109,15 @@ export const CreateEditModal: React.FunctionComponent<CreateEditModalProps> = (p
                             onChange={ (isChecked) => setEventType(prev => ({...prev, subscriptionLocked: isChecked})) }
                         />
                     </FormGroup>
+                    <FormGroup label='is visible?' fieldId='visible'
```
Comment (gwenneg) [inline admin-console/src/main/webapp/src/components/EventTypes/CreateEditModal.tsx:112]:
```suggestion
                    <FormGroup label='Is visible?' fieldId='visible'
```

File: admin-console/src/main/webapp/src/components/EventTypes/CreateEditModal.tsx [hunk 2]
```diff
@@ -109,6 +109,15 @@ export const CreateEditModal: React.FunctionComponent<CreateEditModalProps> = (p
...
+                        helperText='Should the event type be visible by users?'>
```
Comment (gwenneg) [inline admin-console/src/main/webapp/src/components/EventTypes/CreateEditModal.tsx:113]:
```suggestion
                        helperText='Should the event type be visible in the public UI?'>
```

File: admin-console/src/main/webapp/src/pages/ApplicationPage.tsx [hunk 3]
```diff
@@ -107,7 +107,7 @@ export const ApplicationPage: React.FunctionComponent = () => {
     const createEventType = () => {
         setShowModal(true);
         setIsEdit(false);
-        setEventTypes({});
+        setEventTypes({visible:true});
```
Comment (gwenneg) [inline admin-console/src/main/webapp/src/pages/ApplicationPage.tsx:110]: What is the effect of that line?

Comment (g-duval) [reply admin-console/src/main/webapp/src/pages/ApplicationPage.tsx:110]: It defines the event type default value on creation flow.

Comment (gwenneg) [reply admin-console/src/main/webapp/src/pages/ApplicationPage.tsx:110]: So `visible` will be true by default and the other boolean values will be false?

**Pull request merged**